src/mgsocket.h

```diff
@@ -12,6 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#ifdef MGCLIENT_ON_APPLE
+#include <arpa/inet.h>
```

Comment:
`#include "apple/mgcommon.h"` is also applicable here.

Reply:
It's not. Interface depends on some of these includes and then "include resolution" is not trivial anymore.